index.html

```diff
@@ -69,124 +69,73 @@ <h1 class="main-title"> I'm Joaquin. <br>
+          <span><i class="fa fa-circle"></i></span>
+          <span class="year">2015</span>
+        </div>
+        <img src="./img/Snapshoot-Portfolio-popup.svg" alt="Snapshoot-Portfolio" class="img-popup">
+        <img src="./img/snapshoot-portfolio-mobile-popup.svg" alt="mobile-popup" class="img-popup-mobile">
+        <div class="modal-container"> 
+          <div class="information-popup">
+            <p class="paragraph-popup">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              <br>
+              <br>
+              <br>
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+            </p>
+            <p class="paragraph-popup-mobile">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+            </p>
           </div>
-          <ul class="tags">
-            <li class="tag">html</button>
-            <li class="tag">css</button>
-            <li class="tag">javascript</button>
-          </ul>
-          <div class="ref">
-            <button>See project</button>
+          <div class="tags-button">
+            <ul class="tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <ul class="tags-hidden">
+              <li class="tag">Ruby</li>
+              <li class="tag">Bootstrap</li>
+            </ul>
+            <hr>
+            <div class="btn-block">
+              <a href="https://joaquin123455.github.io/microverse-portfolio/" target="_blank" rel="noopener" class="btn-a">
+              <button class="btn-popup-left" type="button"> See Live <img class="img-btn"
+                  src="./img/live-popup.svg" alt="Button of project "> </button></a>
+              <a href="https://github.com/Joaquin123455" target="_blank" rel="noopener" class="btn-a">
+              <button class="btn-popup-right" type="button">See Source <img class="img-btn"
+                  src="./img/github-popup.svg" alt="Button of repository"> </button></a>
+            </div>
           </div>
         </div>
+      </div>
```

Comment:
- [ ] In the Figma template there is just one example of the pop-up window, but logically there should be a different pop-up window for each work with a different title and picture, additionally, we could use JavaScript to populate the modal window as well done with the work cards. Please 🙏🏽 complete the rest of pop-up windows for each work card for a better UX.